src/sbuf/MemBlob.cc

```diff
@@ -124,6 +124,17 @@ MemBlob::append(const char *source, const size_type n)
     ++Stats.append;
 }
 
+void
+MemBlob::moveToStartAndKeep(const size_type offset, const size_type n)
```

Comment:
commit f9f0992fb

I used 'pos' as parameter name instead of 'n'. The 'n' implies a "number of".

Reply:
> The 'n' implies a "number of".

If we call the method "shiftLeft" we should call its parameter `n` rather than `pos`. The parameter _is_ the number of bytes/times/positions we want to shift (left). See, for example, the same parameter in [std::shift_left](https://en.cppreference.com/w/cpp/algorithm/shift)

I know that you were thinking in terms of "position" before, but that (caller-based) thinking does not match the general-purpose method we are adding here. The end _result_ is the same, of course.

Reply:
ok